pkg/admission/admission_test.go

```diff
@@ -316,6 +316,8 @@ func api() *Admission {
 	a := New(
 		slog.New(slog.NewTextHandler(os.Stdout, &slog.HandlerOptions{
 			// slog level math.MaxInt means no logging
+			// We would like to use the slog buil-in No-op level once it is available
+			// More: https://github.com/golang/go/issues/62005
```

Comment:
As I said in the previous PR. Just improving the documentation using this finding -> https://github.com/prometheus-operator/prometheus-operator/pull/6832#discussion_r1715140429